scripts.d/support/create_catalog_ng.sh

```diff
@@ -22,8 +22,10 @@ function addToCatalog()
 
     case $name in
         localhost)
+            nbn=$(LANG=C lscpu| grep 'CPU(s):' | sort -u | head -n +1 |awk -F ":" '{print $2}'| tr -d ' ')
```

Comment:
Let's simplify a bit...
`LANG=C lscpu | grep '^CPU(' | cut -d':' -f2`

Reply:
For the record in SCBI we have:

`SCBI_JOBS=${SCBI_JOBS:-$(grep -c ^processor /proc/cpuinfo 2>/dev/null || sysctl -n hw.ncpu)}`